cdk/lib/constructs/api.ts

```diff
@@ -44,6 +67,21 @@ export class BackendApi extends cdk.Construct {
 
     this.addPrivateEndpoints(apiIntegration, authorizer);
     this.addPublicEndpoints(publicApiIntegration);
+
+    const key = this.api.addApiKey("PerformanceDashboardApiKey");
```

Comment:
I would call this key `IngestApiKey` or `PerfDashIngestApiKey` because it's really just for the ingest api. It won't work for the other apis.